registry/storage/registry.go

```diff
@@ -203,10 +203,7 @@ func (repo *repository) Named() reference.Named {
 }
 
 func (repo *repository) Tags(ctx context.Context) distribution.TagService {
-	tags := &tagStore{
-		repository: repo,
-		blobStore:  repo.registry.blobStore,
-	}
+	tags := NewStore(repo, repo.registry.blobStore)
```

Comment:
we should pass `ctx` here so we can extract logger out of it and use it to log whatever we need to log insetad of importing `logrus` i.e.  `NewStore` would contain something like this:

```Go
logger := context.GetLogger(ctx).Infof("Using default val: %d", foo)
```